service-map/src/main/java/fi/nls/oskari/map/layer/formatters/LayerJSONFormatterWFS.java

```diff
@@ -18,6 +18,9 @@ public JSONObject getJSON(OskariLayer layer,
 
         final JSONObject layerJson = getBaseJSON(layer, lang, isSecure, crs);
         JSONHelper.putValue(layerJson, KEY_ISQUERYABLE, true);
+        // getBaseJSON adds these but model builder isn't using them. Frontend uses DescribeLayer response
+        layerJson.remove(KEY_OPTIONS);
+        layerJson.optJSONObject(KEY_ATTRIBUTES).remove("data");
```

Comment:
We might want to keep this still in the layer response because of https://github.com/oskariorg/oskari-frontend/blob/master/bundles/mapping/mapmodule/AbstractMapModule.js#L2346-L2360

Reply:
removed